setup.py

```diff
@@ -15,7 +15,7 @@
     'pytz',
     'pandas',
     'jsonpath_ng',
-    'bpanonymize @ git+ssh://kcho@github.com/AMP-SCZ/dpanonymize@v0.1#egg=bpanonymize'
+    'bpanonymize@git+https://github.com/AMP-SCZ/dpanonymize@v0.1#egg=bpanonymize'
```

Comment:
Yeah this is expected to work since I have made the repo public.

Reply:
Now `dpanonymize` uploaded to pypi. I'll need to fix here and there for the `dpanonymize`, but at least the installation process for `dpanonymize` work fine linked in `lochness` installation.